__tests__/app.test.js

```diff
@@ -124,6 +124,45 @@ describe('Topics', () => {
 
 describe('Articles', () => {
     
+    describe('GET /api/articles', () => {
+        
+        test('should respond with a 200 status & an array of articles', () => {
+            return request(app)
+            .get(`/api/articles`)
+            .expect(200)
+            .then((response) => {
+                expect(response.body).toHaveProperty('articles');
+                expect(response.body.articles).toBeInstanceOf(Array);
+            });
+        });
+
+        test('should respond array of articles (of correct length) with appropriate properties', () => {
+            return request(app)
+            .get(`/api/articles`)
+            .expect(200)
+            .then((response) => {
+                const articles = response.body.articles;
+                // J.D: Please be aware that if you modify the test data, this test may fail
+                expect(testData.articleData).toHaveLength(13);
```

Comment:
Is this line asserting that the testData isn't changed? It may be worth documenting why